py/ringbuf.c

```diff
@@ -59,6 +60,18 @@ int ringbuf_get(ringbuf_t *r) {
     return v;
 }
 
+int ringbuf_get16(ringbuf_t *r) {
```

Comment:
In the long run, the 16-bit stuff could be factored out so it doesn't take space in a non-LTO build.